examples/joint_inference/helmet_detection_inference/README.md

```diff
@@ -105,42 +88,71 @@ spec:
   edgeWorker:
     model:
       name: "helmet-detection-inference-little-model"
-    nodeName: $EDGE_NODE
     hardExampleMining:
       name: "IBT"
       parameters:
         - key: "threshold_img"
           value: "0.5"
         - key: "threshold_box"
           value: "0.5"
-    workerSpec:
-      scriptDir: "/code/little-model"
-      scriptBootFile: "little_model.py"
-      frameworkType: "tensorflow"
-      frameworkVersion: "1.15"
-      parameters:
-        - key: "input_shape"
-          value: "416,736"
-        - key: "video_url"
-          value: "rtsp://localhost/video"
-        - key: "all_examples_inference_output"
-          value: "/home/data/model/little-model/output"
-        - key: "hard_example_cloud_inference_output"
-          value: "/home/data/model/little-model/hard_example_cloud_inference_output"
-        - key: "hard_example_edge_inference_output"
-          value: "/home/data/model/little-model/hard_example_edge_inference_output"
+    template:
+      spec:
+        nodeName: $EDGE_NODE
+        containers:
+        - image: kubeedge/sedna-example-joint-inference-helmet-detection-little:v0.1.0
+          imagePullPolicy: IfNotPresent
+          name:  little-model
+          env:  # user defined environments
+          - name: input_shape
+            value: "416,736"
+          - name: "video_url"
+            value: "rtsp://localhost/video"
+          - name: "all_examples_inference_output"
+            value: "/data/output"
+          - name: "hard_example_cloud_inference_output"
+            value: "/data/hard_example_cloud_inference_output"
+          - name: "hard_example_edge_inference_output"
+            value: "/data/hard_example_edge_inference_output"
+          ports:  # user defined ports
+            - containerPort: 80
+              protocol: TCP
+          resources:  # user defined resources
+            requests:
+              memory: 64M
+              cpu: 100m
+            limits:
+              memory: 2Gi
+          volumeMounts:
+            - name: localvideo
```

Comment:
localvideo is useless, because you use the rtsp video url instead of local video